example/README.md

```diff
@@ -0,0 +1,171 @@
+# Wattsight API python library usage example
+An introductory example of using Wattsight API python library for searching, fetching, and plotting Wattsight data series. A number of use cases is presented in order to introduce different data structures.
+
+## Dependencies
+
+[Wattsight API python library](https://github.com/wattsight/wapi-python/)   
+[Jupyter (iPython) Notebooks](http://jupyter.org/)   
+[Pandas Data Analysis library](http://pandas.pydata.org/)   
+[Plotly Charting library](https://plot.ly/)   
+[World Timezone Definitions for Python](http://pytz.sourceforge.net/)   
+
+## Installation of Wattsight API python library
+
+1. Using curl
+
+	```command
+	curl -LOk https://github.com/wattsight/wapi-python/archive/wapi-python-0.0.7.zip
+	unzip wapi-python-0.0.7.zip
+	cd wapi-python-wapi-python-0.0.7
+	python setup.py install
+	```
+
+2. Using pip
+
+	```command
+	pip install git+https://github.com/wattsight/wapi-python@wapi-python-0.0.7
+	```
+
+## Implementation
+Complete working code can be found in the attached Jupyter Notebook file.
+
+1. Established a session using client cridentials. 
+
+	This requires having an OAuth client, which can be created at https://auth.wattsight.com/account/oauth-clients.
+
+	```python
+	>>> session = wapi.Session(client_id='client id', client_secret='client secret')
```

Comment:
You don't want to teach people to use the ini file variant?  I would expect that to be easier to deal with, and hopefully have people not hardcode clientids/secrets in their code.

Reply:
you can't use config file with only client_id and client_secret specified. and i do not want to look into urlbase, auth_type, and so on.

Reply:
Hmm..  Sounds like a mis-feature in the auth code.  It should really allow things with proper defaults to not be specified in the ini file.